pallets/gear/src/internal.rs

```diff
@@ -631,7 +649,7 @@ where
             let hold = delay_hold.min(maximal_hold);
 
             // Locking funds for holding.
-            GasHandlerOf::<T>::lock(dispatch.id(), hold.lock())
+            GasHandlerOf::<T>::lock(LockIdentifier::DispatchStash, dispatch.id(), hold.lock())
```

Comment:
HoldBound may contain LockIdentifier to be consistent with usage within GasHandler::lock 
```suggestion
            GasHandlerOf::<T>::lock(dispatch.id(), hold.lock_type(), hold.lock_amount())
```